src/toolsData.js

```diff
@@ -299,6 +300,14 @@ export default {
         url: "https://aws.amazon.com/appsync/",
         github: "https://github.com/awslabs/aws-mobile-appsync-sdk-js",
         language: JavaScript
+      },
+      {
+        name: "TypeGraphQL",
+        description:
+          "TypeGraphQL is a modern framework for building GraphQL API in Node.js. It allows to create GraphQL schema and resolvers using only classes and decorators.",
```

Comment:
Please change to the following (I tweaked the grammar a bit): ` "TypeGraphQL is a modern framework for building GraphQL APIs in Node.js. It allows creating the GraphQL schema and resolvers using only classes and decorators."`

Reply:
It's fine! Thank you for the contribution.